diffy/plugins/diffy_aws/ssm.py

```diff
@@ -122,18 +122,18 @@ def retry_pending(exception) -> bool:
 def poll(command_ids: dict, **kwargs) -> dict:
     """Query the SSM endpoint to determine whether a command has completed.
 
-    :returns: List. Results of command(s)
```

Comment:
Ah this may be part of my misunderstanding, is this passing back a list of results with no commands? Or a dict of resultof[command_string] = result_json?

Reply:
Ok looked more closely at the documented format, dict of lists of dicts.